app/models/event/activation.rb

```diff
@@ -0,0 +1,36 @@
+class Event::Activation < ApplicationRecord
+  belongs_to :event, required: true
```

Comment:
`required: true` is deprecated since Rails 5.0, we can use `optional: false` instead. Since we did not [change the default configuraion](http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#active-record-belongs-to-required-by-default-option) for our app yet, a deprecation warning might actually be a good thing, though. 

> :required
When set to true, the association will also have its presence validated. This will validate the association itself, not the id. You can use :inverse_of to avoid an extra query during validation. NOTE: required is set to true by default and is deprecated. If you don't want to have association presence validated, use optional: true.